src/Discovery/Discovery.css

```diff
@@ -569,6 +569,7 @@
 
 .discovery-modal__field {
   display: flex;
+  white-space: pre-wrap;
```

Comment:
pre-wrap "Sequences of white space are preserved. Lines are broken at newline characters, at [<br>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br), and as necessary to fill line boxes." https://developer.mozilla.org/en-US/docs/Web/CSS/white-space